api/src/controllers/events.ts

```diff
@@ -1 +1,52 @@
-// events.ts
+import { errors as elasticErrors } from "@elastic/elasticsearch";
+import { RequestHandler } from "express";
+import asyncHandler from "express-async-handler";
+import { Clients, Displayable } from "../types";
+import { PaginationQueryParameters } from "./pagination";
+import { Config } from "../../config";
+import { HttpError } from "./error";
+import { ResultList } from "../types/responses";
+import { resultListResponse } from "../helpers/responses";
+
+type QueryParams = {
+  query?: string;
+  sort?: string;
+  sortOrder?: string;
+  aggregations?: string;
+  "contributors.contributor"?: string;
+  "publicationDate.from"?: string;
+  "publicationDate.to"?: string;
+  format?: string;
+} & PaginationQueryParameters;
```

Comment:
Maybe more for @agnesgaroux to review the list, I think we were skipping contributors for now? I know we're doing a very generic version of it all for now, so might be a nitpick.

Reply:
Not a nitpick - is a valid point! Can remove them for now :)

Reply:
Just compare with what's currently there on the work Agnès did I reckon

Reply:
I started thinking about filters when I was building the EventDocument transformer but realised it wasn't really a dev decision, so we don't have aggregations or filters in event documents yet
I'd just keep 
```
  query?: string;
  sort?: string;
  sortOrder?: string;
  aggregations?: string;
```
`format?` could stay as well because it's very likely something we'll want to filter by, OTOH `publicationDate` can def go as it's an article concept and is not relevant to events

Reply:
Reflects this now - have kept `format?` if it'll be used eventually, but happy to remove if preferred